holmscan/cli.py

```diff
@@ -77,10 +175,34 @@ def parse_cli():
     log.debug("Setting verbose level: {0}".format(cli_args["--verbose"]))
     log.debug("Arguments from CLI: \n{0}".format(pformat(cli_args)))
 
-    if cli_args["<command>"] == "kaboom":
-        sub_args = docopt(
-            eval("{sub}_args".format(sub=cli_args["<command>"])), argv=argv
-        )
+    if cli_args["<command>"] == "net":
+        sub_args = docopt(net_args, argv=argv)
+        if sub_args["asset"]:
+            sub_args = docopt(net_asset_args, argv=argv)
+        elif sub_args["profile"]:
+            sub_args = docopt(net_profile_args, argv=argv)
+        elif sub_args["scan"]:
+            sub_args = docopt(net_scan_args, argv=argv)
+        #elif sub_args["schedule"]:
+        #    sub_args = docopt(net_schedule_args, argv=argv)
+        else:
+            extras(
```

Comment:
This extras call seems to be present multiple times, maybe abstract into a separate call, or look at maybe having it as a globally "else" case that it defaults out to. I am thinking like
```
def print_help(base_args):
  extras(True, holmsan.__version__, [Option('-h', '--help', 0, True)], base_args
```